### PR TITLE
SQL Query Prompt update + added _execute method for SQLDatabase

### PR DESCRIPTION
**Pull request by keenborder786**
- Description: This pull request (PR) includes two minor changes:

1. Updated the default prompt for SQL Query Checker: The current prompt does not clearly specify the final response that the LLM (Language Model) should provide when checking for the query if `use_query_checker` is enabled in SQLDatabase Chain. As a result, the LLM adds extra words like "Here is your updated query" to the response. However, this causes a syntax error when executing the SQL command in SQLDatabaseChain, as these additional words are also included in the SQL query. 

2. Moved the query's execution part into a separate method for  SQLDatabase: The purpose of this change is to provide users with more flexibility when obtaining the result of an SQL query in the original form returned by sqlalchemy. In the previous implementation, the run method returned the results as a string. By creating a distinct method for execution, users can now receive the results in original format, which proves helpful in various scenarios. For example, during the development of a tool, I found it advantageous to obtain results in original format rather than a string, as currently done by the run method.

- Tag maintainer: @hinthornw

### CODE REVIEW

**Comment by keenborder786** [comment]
@hinthorn any update or feedback?

**Comment by keenborder786** [comment]
Updates @hinthorn

**Comment by maaz1m** [comment]
This would be a massive improvement in my current workflow. @hinthornw can you please look into this PR?

File: libs/langchain/langchain/tools/sql_database/prompt.py [hunk 1]
```diff
@@ -11,4 +11,8 @@
 - Casting to the correct data type
 - Using the proper columns for joins
 
-If there are any of the above mistakes, rewrite the query. If there are no mistakes, just reproduce the original query."""
+If there are any of the above mistakes, rewrite the query. If there are no mistakes, just reproduce the original query.
+
+Output the final SQL query only.
+
+SQL Query: """
```
Comment (baskaryan) [inline libs/langchain/langchain/tools/sql_database/prompt.py:18]: @keenborder786 updated the prompt slightly, what do you think?

Comment (keenborder786) [reply libs/langchain/langchain/tools/sql_database/prompt.py:18]: @baskaryan please merge.

Comment (keenborder786) [reply libs/langchain/langchain/tools/sql_database/prompt.py:18]: @baskaryan please merge this, this is really important. Thank you.

**Comment by baskaryan** [comment]
thanks @keenborder786!

**Pull request merged**